### ISSUE TITLE
Actualize Test branch and Synchronize Master

### ISSUE THREAD

**Issue by orestotel** [orestotel/hacktoberfest-ProjectCPlusPlus#11]
The ___Master___ branch needs to be merged into the test branch because there were a lot of files added since the ___Test___ branch was last time edited, so I decided to make it relevant and actualize it. It cannot be merged automatically, because of the conflicts created by #5 pull request, which fixed an important issue #2 . 

So now I will merge ___Master___ into ___Test___ and leave all the useful changes in ___Test___ branch, and then merge Test branch into Master, again, to actualize the #5 commit.

TL;DR - merge ___Master___ into ___Test___, then merge ___Test___ into ___Master___ and resolve all the conflicts.

**Comment by orestotel** [comment]
#19 #18 #17 fixes the issue
